test/source/mock/google/google-endpoints.ts

```diff
@@ -58,7 +58,13 @@ export type MockUserAlias = {
 export interface GoogleConfig {
   contacts?: string[];
   othercontacts?: string[];
-  aliases?: Dict<MockUserAlias[]>;
+  aliases?: Record<
+    string,
+    {
+      primarySignature?: string;
+      list?: MockUserAlias[];
+    }
+  >;
```

Comment:
Tom mentioned in issue that he doesn't want to have exception for `flowcrypt.compatiblity@gmail.com` aliases.
So I changed aliases type to like this.

@sosnovsky Do you agree with this change?
Or you have better idea?
> I would rather not have an exception for this:
> 
>       // If the account is a compatibility account, return specific aliases.
>       // This account is used in hundreds of tests, so we handle it separately to avoid duplicating the code.
>       if (acct === 'flowcrypt.compatibility@gmail.com') {
>

Reply:
Maybe it's possible to make `primarySignature` property of `GoogleConfig` and then `aliases` type will be `Dict<MockUserAlias[]>` as before?

Reply:
Yeah, That sounds better